config/initializers/devise_monkeypatch.rb

```diff
@@ -0,0 +1,24 @@
+# Changing the value for "domain" in each context instead of using the one set on boot.
+# This allows changing domain settings without restarting app.
+module Devise
+  module Controllers
+    module Rememberable
+      # We need to use SiteConfig.app_domain instead of default Rails config on boot
+      def remember_cookie_values(resource)
+        options = { httponly: true }
+        options.merge!(forget_cookie_values(resource))
+        options.merge!(
+          value: resource.class.serialize_into_cookie(resource),
+          expires: resource.remember_expires_at,
+          domain: ".#{SiteConfig.app_domain}",
```

Comment:
At some point (a separate PR?) we may want to extract this out rather than appending `.` in front of the app domain. Too scared/scared from that double-cookie domain scenario last week and afraid we might accidentally forget to add this at some point 😬

Reply:
I think we're safe from getting this wrong because the cookie isn't going to be set if the request doesn't match the request domain.

But 1000% understand the scarring issue. It's the right concern. I'm pretty sure this is safe, but we could follow up with more safeguards and refactors on this all in future work.